Update guideline for WebApplicationInfo.Id (#4029)
MicrosoftDocs/msteams-docs: msteams-platform/bots/how-to/authentication/auth-aad-sso-bots.md

**webApplicationInfo** is the parent of the following elements:

> * **id** - The client ID of the application. This is the application ID that you obtained as part of registering the application with AAD. It is recommended that this Application ID not be shared between multiple Teams apps and a new AAD app be created for each application manifest that uses `webApplicationInfo`.

Reviewer: Suggest rephrasing it to:

```suggestion
* **id** - The client ID of the application. This is the application ID that you obtained as part of registering the application with AAD. Don't share this Application ID with multiple Teams apps. Create a new AAD app for each application manifest that uses `webApplicationInfo`.
```
